### ISSUE TITLE
Issue after this update

### ISSUE THREAD

**Issue by domleg** [bmgjet/PirateNPCs#6]
Error while compiling: PirateNPC.cs(3613,47): error CS0120: An object reference is required to access non-static member `BaseMountable.TryFireProjectile(StorageContainer, Rust.AmmoTypes, UnityEngine.Vector3, UnityEngine.Vector3, BasePlayer, float, float, out ServerProjectile)'

Trying to fix this but not sure what is wrong. BaseMountable.TryFireProjectile is a public bool in Assembly-CSharp...

I see in an older dll they had it like this public static bool TryFireProjectile and now it's like this public bool TryFireProjectile

**Comment by mbedda** [comment]
Yes because it's not static now, add this line before the if statement:
`BaseMountable mount = npcs[0].GetMounted();`
then do `mount.TryFireProjectile`

**Comment by domleg** [comment]
Yep! Already figured it out thanks!
